doc/README.android.md

```diff
@@ -14,33 +13,35 @@ shown), the following controls are available:
 * long tap: dig node
 * touch shown buttons: press button
 * Buttons:
-** left upper corner: chat
-** right lower corner: jump
-** right lower corner: crouch
-** left lower corner: walk/step...
-   left up right
-       down
-** left lower corner: display inventory
+
+1. left upper corner: chat
+2. right lower corner: jump
+3. right lower corner: crouch
+4. left lower corner (Joystick): walk/step...
+5. left lower corner: display inventory
 
 When a menu or inventory is displayed:
 * double tap outside menu area: close menu
 * tap on an item stack: select that stack
 * tap on an empty slot: if you selected a stack already, that stack is placed here
 * drag and drop: touch stack and hold finger down, move the stack to another
   slot, tap another finger while keeping first finger on screen
-  --> places a single item from dragged stack into current (first touched) slot
-
+  --> places a single item from dragged stack into current (first touched) slot. If a stack is selected, the stack will be split as half and one of the splitted stack will be selected
 Special settings
 ----------------
 There are some settings especially useful for Android users. Minetest's config
-file can usually be found at /mnt/sdcard/Minetest.
+file can usually be found at:
+
+* old version: ``/sdcard/Minetest/`` or ``/storage/emulated/0/`` if stored in device, if stored in SD card: ``/storage/emulated/(varying folder name)/``
+* new version: ``/sdcard/Android/data/net.minetest.minetest/`` or ``/storage/emulated/0/Android/data/net.minetest.minetest/`` if stored in device, if stored in SD card: ``/storage/emulated/(varying folder name)/Android/data/net.minetest.minetest/``
+* NOTE: since Android 11 you can't access Android folder with normal methods
```

Comment:
This is false, you're able to access it up to at least Android 12 using the method listed here: https://forum.minetest.net/viewtopic.php?f=18&t=27684&p=405413#p405413

You're of course also able to access it through ADB on any version as well.

Reply:
i changed the text while waiting for answer, im reading Android doc because i dont Android 11+ device

Reply:
Officially speaking the method listed in the forum thread is a bug, but it's still a method to access the engine's user data if your file manager app supports it.

Reply:
ok i removed it